.cdl/development/schema-registry.toml

```
@@ -1,4 +1,5 @@
 input_port = 50101
+import_file = "bare/setup/schema-registry/initial-schema.kafka.json"
```

We need a nice way to patch those entries on horust level, maybe using `patch`? Or creating a more sophisticated `overrides` mechanism?

---

For now either
```suggestion
#import_file = "bare/setup/schema-registry/initial-schema.kafka.json"
```
or
```suggestion
```

---

That's y I would prefer for everyone to create their own set of configs. I dunno, maybe Friday meeting will shed some light onto this. I'ma note that this is an use case.

---

its problematic for horust deployment to have some additional steps before launching, config management left on user side is a problem since  you have technically have to make matrix consisting of (deployment * version * variant).
I was under the impression we will be able to make a directory with variants in the cdl repo so that we can patch it up against deployment automatically during the start.
I dunno, we will have to think something out.